plugins/GSdx/GSDeviceOGL.cpp

```diff
@@ -247,7 +247,7 @@ bool GSDeviceOGL::Create(GSWnd* wnd)
 	m_convert.cb = new GSUniformBufferOGL(g_convert_index, sizeof(ConvertConstantBuffer));
 	// Upload once and forget about it
 	ConvertConstantBuffer cb;
-	cb.ScalingFactor = GSVector4i(theApp.GetConfig("nativeres", 0) ? 1 : theApp.GetConfig("upscale_multiplier", 2));
+	cb.ScalingFactor = GSVector4i(theApp.GetConfig("upscale_multiplier", 1)==1 ? 1 : theApp.GetConfig("upscale_multiplier", 2));
```

Comment:
custom res was 6 (now 7) it then later on gets changed in code to 1, but the config should always return its actual value.